packages/shared/src/game/ThreatDragonModel.ts

```diff
@@ -332,7 +332,8 @@ type CellType =
   | 'tm.Store'
   | 'tm.Actor'
   | 'tm.Flow'
-  | 'tm.Boundary';
+  | 'tm.Boundary'
+  |'tm.BoundaryBox';
```

Comment:
For consistency, please add a whitespace here

Reply:
done